src/Components/Intro/Intro.js

```diff
@@ -1,40 +1,54 @@
 import React from "react";
 import AnchorLink from 'react-anchor-link-smooth-scroll';
 
+const ciphers = [
+    "atbash",
+    "caesar",
+    "vigenere"
+]
+const cipherPretty = [
+    "Atbash",
+    "Caesar",
+    "Vigenere"
+]
+
 function Intro(props) {
     return (
         <div id="intro">
             <div className="container">
-                <div className="column">
+            <div className="columns is-vcentered container">
+                <div className="column is-9">
                     <div className="columns is-vcentered container">
-                        <div className="is-size-3 content-custom column container">
-                        <br />
-                        What is encryption? 
+                        <br/>
+                        <div className="is-size-4 content-custom column container">
+                        
                         How does ZHOFRPH WR JUDYLWB IDOOV turn into WELCOME TO GRAVITY FALLS? 
                         And how is that related to computers??  
-                        </div>
-                    </div>
-                    <div className="columns is-vcentered container">
-                        <div className="is-size-3 content-custom column container">
+                        <br/><br/>
+                        <span role="img" aria-label="key">🔑</span>   <span role="img" aria-label="computer">💻</span>   <span role="img" aria-label="keywithlock">🔐</span>
+                        <br/><br/>
                         Computers use cryptography all the time. 
                         Humankind has encrypted messages since as early as 1500 B.C. 
                         Let’s start from the beginning... 
-                        <br /><br />
                         </div>
-                    </div> 
+                    </div>
+                </div>
+                <div className="column row_space"> 
+                    {ciphers.map((cipher, i) => {
+                        return(
+                            <div className="row_space">
+                                <AnchorLink offset={-5} href={"#"+cipher}>
+                                    <button class="button is-static is-large is-family-secondary has-text-weight-bold">
+                                        {cipherPretty[i]} Cipher
+                                    </button>
+                                </AnchorLink>

```

Comment:
A good way of handling this would be:
```suggestion
                            <div className="row_space">
                                <AnchorLink offset={-5} href={"#"+cipher} className="button is-static is-large is-family-secondary has-text-weight-bold">
                                    {cipherPretty[i]} Cipher
                                </AnchorLink>
```

It's kind of cheating, since we are applying the `.button` class to a link, but this is what we did with the [links in the title](https://github.com/uclaacm/cipher-salad/blob/master/src/Components/Title/Title.js#L68).

Reply:
Thanks Leo. In this case I took out the is-static class property so the buttons are still clickable